runtime/jcl/CMakeLists.txt

```diff
@@ -47,6 +56,60 @@ target_include_directories(j9vm_jcl_includes
 )
 
 
+# Generate constant pool files
+if(J9VM_GEN_CP)
+	set(CONST_POOL_OUT_DIR "${j9vm_BINARY_DIR}")
+
+	# List of files generated by J9VMCPTool
+	set(gen_constpool_files)
+	foreach(jcl IN LISTS JCLS)
+		list(APPEND gen_constpool_files "${CONST_POOL_OUT_DIR}/j9vmconstantpool_${jcl}.c")
+	endforeach()
+	if(OMR_HOST_OS STREQUAL "win")
+		set(JAVA_PATH_SEP ";")
+	else()
+		set(JAVA_PATH_SEP ":")
+	endif()
+
+	# Dump the current cmake cache to text file
+	# We could just use CMakeCache.txt, but this way filters out advanced/internal variables and comments
+	add_custom_command(
+		OUTPUT parsed_cache.txt
+		COMMAND "${CMAKE_COMMAND}" -N -L > "${CMAKE_CURRENT_BINARY_DIR}/parsed_cache.txt"
+		WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
+	)
+	# Take our list of jcls, eg "se7_basic;se9;se9_before_b165" and replace ';' with ','
+	# Store the result in jcls_argument
+	string(REPLACE ";" "," jcls_argument "${JCLS}")
+	add_custom_command(
+		OUTPUT
+			${gen_constpool_files}
+			COMMAND "${Java_JAVA_EXECUTABLE}" -cp "$<TARGET_PROPERTY:j9vmcp,JAR_FILE>${JAVA_PATH_SEP}$<TARGET_PROPERTY:objectmodel,JAR_FILE>" com.ibm.oti.VMCPTool.Main
```

Comment:
Can you please fix the indentation here.

Reply:
Besides the indentation of `COMMAND`, I had expected that `OUTPUT` would be on one line like elsewhere, but I won't hold this up for that.